polyaxon/values.yaml

```diff
@@ -424,7 +424,7 @@ postgresql:
   #     postgresUser:
   #     postgresPassword:
   #     postgresDatabase:
-  enabled: true
```

Comment:
Can you update the the comment in [line 421](https://github.com/polyaxon/polyaxon-chart/pull/23/files#diff-43bc6296fbc9a23e12a4cce5c0f46e0eL421)

Reply:
Done 👍